### ISSUE TITLE
App not formatting properly with zoomed view selected for Android (Accessibility)

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#619]
This issue mirrors hyperledger/aries-mobile-agent-react-native#773 which was created for the community to see.

**Comment by knguyenBC** [comment]
Some screen still have cut off buttons. See attached the home screen, contacts, and PIN screen: 
![20221205_170308.jpg](https://images.zenhubusercontent.com/619bd30a068b26108b1bc2fe/0f0fc4d0-a586-4b4e-8e7b-52521619d851)![20221205_170323.jpg](https://images.zenhubusercontent.com/619bd30a068b26108b1bc2fe/c6dfb79f-cdcd-4cb1-81a6-d9d6be135068)![20221205_170421.jpg](https://images.zenhubusercontent.com/619bd30a068b26108b1bc2fe/fedb9d1c-3128-49fb-bbfb-205ad36be203)

**Comment by nodlesh** [comment]
@knguyenBC @cvarjao How is Zoom enabled on Android (which is what this bug references)? I can turn on Magnification but that allows me to pan the screen to get what I need in view. Are the examples above just font size set to extra large? Thanks.

**Comment by knguyenBC** [comment]
There's display size and font size that people can configure. Both are in settings. I had maxed out both settings and they do not allow panning.

**Comment by nodlesh** [comment]
Tested with build 683 on a Samsung Galaxy S10 Android 11. I maxed out the display size and the font size as mentioned by Kim. These issues remain. We should discuss if this is acceptable. ![Screenshot_2023-02-01_at_11_14_04_AM.jpg](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/92cc6b6e-49b1-4d38-aaf5-56919420f9a0)![Screenshot_2023-02-01_at_11_16_14_AM.jpg](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/45d72ec5-5679-4560-ab07-f8a4adeb64fe)![Screenshot_2023-02-01_at_11_20_27_AM.jpg](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/cbee503f-2e68-462d-84fe-7de93d07ea85)![Screenshot_2023-02-01_at_11_21_41_AM.jpg](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/033df9d5-9ac3-49cd-8c9b-96e7fbae2329)![Screenshot_2023-02-01_at_11_22_01_AM.jpg](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/437cf5f3-26c4-484e-8800-bf74a4b72f39)

**Comment by bryce-mcmath** [comment]
I was not able to test on a Galaxy S10 like Sheldon used, but most of these issues have been worked on. However, the notification count badge text has not been adjusted as it is a limitation of react-navigation, see here: https://github.com/react-navigation/react-navigation/issues/10219

**Comment by knguyenBC** [comment]
@bryce-mcmath ![Screenshot_20230406-132224 (2).jpg](https://images.zenhubusercontent.com/619bd30a068b26108b1bc2fe/47e14082-b43a-4982-bdf3-3b39e1503d77) Missing info icon. I assume it gets pushed to the side with text enlargement.

**Comment by jleach** [comment]
@wadeking98 Did you fix this one already?

**Comment by jleach** [comment]
I tested on a Pixel 6 emulator with Font Size and Display Sizes both set to max. All the screens look fine. Screen shots attached. Could some phones just have poor quality options for this Zoom feature? I don't think it makes sence to set your font size to 92 and expect things to look good.

Some screens could do with a little polishing. The info icon on the chat screen is a little cut off and the "this is taking longer than it should" message when waiting for a credential offer to come through does not scroll as much as it should. Others are off a little also but I think they're quire reasonable.

- [ ] Create PIN
- [ ] Home
- [ ] Settings
- [ ] Credentials
- [ ] Contacts
- [ ] Chat


![Screenshot 2023-05-16 at 5.07.07 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/7f3b1ab7-13e0-459b-8c10-7c58b75d5910)

![Screenshot 2023-05-16 at 4.36.42 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/e956d2c7-da5e-4b35-ac27-cf5dd1da88ad)

![Screenshot 2023-05-16 at 5.13.13 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/ad299ab9-52e5-401a-bd4f-f20ee8439edd)

![Screenshot 2023-05-16 at 4.36.28 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/a55d4b4f-ff1a-4992-b7b8-fae21c6c11a0)

![Screenshot 2023-05-16 at 4.36.14 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/18758da2-4f72-4222-880b-eb6d665d465d)

![Screenshot 2023-05-16 at 5.18.30 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/97296180-a3b8-45c5-8602-70ee9c15daba)

**Comment by knguyenBC** [comment]
The info button in the contacts details page is still a problem with longer names (it gets completely obscured) but overall, the other issues, we can say is good enough. The main goal is to ensure functionality is still there, so as long as the button or text is still somewhat visible and tap-able. It's less about ensuring its still visibly appealing.

**Comment by jleach** [comment]
@knguyenBC The Pixel 6 i tested on has a 6.4" screen size where as the OnePlus 6 I think you used for testing has a slightly larger size of 6.82". I'll fix the contact details info button.
